.evergreen/handle-paths.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+# This script will handle the correct cross-platform absolute
+# paths for a script directory and DRIVERS_TOOLS.  
+# It is meant to be invoked as follows:
+#
+# SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+# . $SCRIPT_DIR/../handle-path.sh
+
+set -o errexit  # Exit the script with error if any of the commands fail
+
+if [ -z "$SCRIPT_DIR" ]; then 
+  echo "Please set $SCRIPT_DIR first"
+  exit 1
+fi
+
+if command -v realpath &> /dev/null
+then
+    SCRIPT_DIR=$(realpath $SCRIPT_DIR)
+else 
+  SCRIPT_DIR="$( cd -- "$SCRIPT_DIR" &> /dev/null && pwd )"
+fi
+if [ "Windows_NT" = "${OS:-}" ]; then # Magic variable in cygwin
+  SCRIPT_DIR=$(cygpath -m $SCRIPT_DIR)
+fi
+
+# Find the DRIVERS_TOOLS by walking up the folder tree until there
+# is a .evergreen folder in the same directory.
+if [ -z "${DRIVERS_TOOLS:-}" ]; then
+  DRIVERS_TOOLS=$(dirname $SCRIPT_DIR)
+  while 1
```

Comment:
```suggestion
  while true
```

This fixes an [observed failure](https://spruce.mongodb.com/task/mongo_c_driver_testazurekms_variant_testazurekms_task_patch_6f4d492aacc31bcfd81e0244456b58de2ef9c84c_659c0c933e8e86c1a4c04154_24_01_08_14_54_13/logs?execution=0) in C driver tasks: `1: command not found`

Reply:
Fixed in the previous commit.  :)